activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+*   Can preload through polymorphic associations (e.g. `Review.includes(media: [:author, :director])` where not all media records have an author or director)
```

Comment:
What do you thin about: "Add support to preload polymorphic association when not all the records have the  requested associations"?

After that we can add a complete example.